{"name": "test_two", "status": "failed", "statusDetails": {"message": "AssertionError: assert False\n +  where False = hasattr('hello', 'check')", "trace": "self = <test_pytest.test_pytest.TestMyClass object at 0x00000272C6ADB648>\n\n    def test_two(self):\n        x = \"hello\"\n>       assert hasattr(x, \"check\")\nE       AssertionError: assert False\nE        +  where False = hasattr('hello', 'check')\n\ntest_pytest\\test_pytest.py:80: AssertionError"}, "attachments": [{"name": "stdout", "source": "efa7471b-0076-46a2-8e3a-a91c0b0a3c6b-attachment.txt", "type": "text/plain"}], "start": 1583068901796, "stop": 1583068901796, "uuid": "65978764-6d52-4089-8884-bb43661c96a1", "historyId": "8afe24c3debe08db4f2232ad9f364a41", "testCaseId": "3b1b357f1ddfe4f952a75942e4189695", "fullName": "test_pytest.test_pytest.TestMyClass#test_two", "labels": [{"name": "parentSuite", "value": "test_pytest"}, {"name": "suite", "value": "test_pytest"}, {"name": "subSuite", "value": "TestMyClass"}, {"name": "host", "value": "luoluo-PC"}, {"name": "thread", "value": "14108-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_pytest.test_pytest"}]}